server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -88,18 +171,18 @@ object CORS {
             // TODO model me
             "Access-Control-Allow-Origin" -> origin,
             // TODO model me
-            "Access-Control-Max-Age" -> config.maxAge.toString
+            "Access-Control-Max-Age" -> config.maxAge.toSeconds.toString
           )
       }
 
       def allowCORS(origin: Header.Raw, acrm: Header.Raw): Boolean =
         (config.anyOrigin, config.anyMethod, origin.value, acrm.value) match {
           case (true, true, _, _) => true
           case (true, false, _, acrm) =>
-            config.allowedMethods.exists(_.contains(acrm))
+            config.allowedMethods.exists(_.find(_.toString() === acrm).nonEmpty)
```

Comment:
This call, and the one below it, look off to me.  Don't we need to parse the ACRM header and compare methods?

Reply:
For the old implementation (before my commits), doesn't ``allowedMethods`` hold the method names directly?
I might be really misunderstanding this still, but wouldn't calling ``Method#name`` (or in this case ``toString``) be the same?

Reply:
Oh, I see.  I was thinking of the ACRM header as having multiple values, but it's just one.  Yes, I think this logic is correct.